leanpkg/leanpkg/main.lean

```diff
@@ -69,12 +63,12 @@ def init_gitignore_contents :=
 
 def init_pkg (n : string) (from_new : bool) : io unit := do
 write_manifest { name := n, path := "src", version := "0.1" } leanpkg_toml_fn,
-src_ex ← dir_exists "src",
```

Comment:
Can you also check that `src` is a directory? It does not seem like `io.fs.exist` can tell the difference between files and directory.

Reply:
I don't think this is much of a functional change since `leanpkg` would totally break if there was a file there called `src` anyway, but fixed. :)